snaps/features/custom-name-resolution.md

```diff
@@ -28,6 +23,18 @@ For example, to resolve Ethereum Mainnet domains, add the following to your Snap
 }
 ```
 
+If you're only targetting specific TLDs or schemes, you can use the `matchers` property to reduce
```

Comment:
I think showing an example of the `schemes` property would be good and also some language giving an example of what sort of input `tlds` & `schemes` would match on e.g. `hassan.eth` & ` farcaster:hassan` respectively.